strapi/strapi
# [DEITS] Misc - Cleanup / Fixes

### Why is it needed?

- Fix light issues regarding streaming (file source provider)
- Remove console log from the engine
- Added decryption capabilities

### How to test it?

- yarn strapi export (with encryption & compression)
- Create a transfer engine with the local file source provider as a source & see the content being streamed (decompressed & decrypted)
